app/javascript/react/components/ReviewContainer.js

```diff
@@ -17,12 +18,15 @@ const ReviewContainer = ({ reviews }) => {
 
   return (
     <Fragment>
-      <h2 className="large-offset-2"> Reviews </h2>
+      <div className="grid-x review-header">
+        <h2 className="large-offset-2"> Reviews </h2>
+        <Link className="button" to={`/movies/${props.match.params.id}/reviews/new`}>Add a Review</Link>
```

Comment:
along with the redirect you have for visiting the new review form, consider the user experience here to check if there is a currently signed in user and if there's not display a message/button to let them know they need to first sign in